spec/workers/source_check_worker_spec.rb

```diff
@@ -151,8 +151,8 @@
       CollectionMailer.stub(:collection_status).with('name', 'up')
     end
 
-    it 'should suppress the collection' do
-      RestClient.should_receive(:put).with("#{ENV['API_HOST']}/harvester/sources/#{source.id}", source: { status: 'active' }, api_key: ENV['HARVESTER_API_KEY'])
+    it 'should activate the collection and set the status_updated_by as LINK CHECKER' do
```

Comment:
optional, but we should use new syntax where we can, even if it's against the format of the existing file.  Gradually we will clean it all up

Reply:
done - thanks